core/services/vrf/proof/seed.go

```diff
@@ -53,6 +53,7 @@ type PreSeedDataV2Plus struct {
 	CallbackGasLimit uint32
 	NumWords         uint32
 	Sender           common.Address
+	NativePayment    bool
```

Comment:
Trying to understand the big picture - say we release a new V2+ contract, would we be able to use this object for the first V2+ contract? Assuming there's some new parameter, e.g `V2Parameter uint32`? I think it should be fine because the final seed is always onchain preseed + blockhash, but wanted to make sure.

Reply:
Looked into this further and changed it so that `requestCommitment` now has `extraArgs` instead of `nativePayment`. This means that `requestCommitment` passed into `fulfillRandomWords` is now `calldata` instead of `memory`, which I think is fine. This allows us to use the same struct `PreSeedDataV2Plus` for future versions of v2 plus